tests/nn/model_parallel/commons.py

```diff
@@ -99,10 +113,20 @@ def spawn_for_all_world_sizes(test_func, world_sizes=get_world_sizes(), args=[])
         mp.spawn(test_func, args=(world_size, *args), nprocs=world_size, join=True)
 
 
-def helper(rank, world_size, func, args):
+def helper(rank, world_size, func, args, error_queue):
```

Comment:
Maybe a better name and a comment. Ditto for fixer. The terse names make understanding the code challenging.